css/components/app-badges.scss

```diff
@@ -0,0 +1,59 @@
+.layout-app-badges{
+  width: 100%;
+  margin-bottom: 0.5em;
+
+  img{
+    height: 65px;
+  }
+  .app_badge_column__left {
```

Comment:
Aangezien hier geen andere styling op gezet wordt (geen default) is het misschien netter en beter leesbaar om het zo te doen:
```
@media ... {
.app_badge_column__left {}
.app_badge_column__right {}
}
```

Reply:
Weghaald